src/sdk/pynni/nni/platform/local.py

```diff
@@ -36,7 +36,7 @@
     os.makedirs(_outputdir)
 
 _nni_platform = os.environ['NNI_PLATFORM']
-if _nni_platform not in ['pai', 'kubeflow', 'frameworkcontroller']:
+if _nni_platform not in ['pai', 'kubeflow', 'frameworkcontroller', 'remote']:
```

Comment:
Since 'remote' mode supports remote logs now, I think we need change to : 
if _nni_platform == 'local':

Or else, it's very easy to forget to add new platform here once adding a new training platform

Reply:
changed to `if _nni_platform == 'local':`